pkg/plugin/probes-handler/probes_handler_test.go

```diff
@@ -0,0 +1,64 @@
+package probeshandler_test
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+
+	probeshandler "github.com/arquillian/ike-prow-plugins/pkg/plugin/probes-handler"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Test liveliness and readiness probes.", func() {
+
+	const (
+		probesEndpoint = "/version"
+		version        = "xxxxxxxx-xxxxxxxxxx"
+	)
+
+	var _ = BeforeSuite(func() {
+		os.Setenv("VERSION", version)
+	})
+
+	var _ = AfterSuite(func() {
+		os.Clearenv()
```

Comment:
`Clearenv` deletes all environment variables, which is something that we don't want to do. There should be probably used `Unsetenv`. Or, which would be cool, at the line 22 get the environment value that is already set and here reset it back.

Reply:
The variable I am trying to access is exposed inside the docker container and is not exposed in the test scope.
>  which would be cool, at the line 22 get the environment value that is already set and here reset it back.

So, the environment variables return empty values and the only variable set here is the one that is explicitly set in the `BeforeSuite` and this is what is cleared up in the `AfterSuite`.

Reply:
> The variable I am trying to access is exposed inside the docker container and is not exposed in the test scope.

when the test is executed then the variable set on line 22 is exposed in the test scope. Then on line 26 all variables  (which means everything that is locally set) are deleted

> So, the environment variables return empty values and the only variable set here is the one that is explicitly set in the BeforeSuite and this is what is cleared up in the AfterSuite.

if I have set the variable `VERSION` for any reason locally, then this test would reset my original value.

Reply:
Resetting the value to the original after suite as discussed offline.